pcdsdevices/pim.py

```diff
@@ -340,6 +340,19 @@ class XPIMFilterWheel(StatePositioner):
     error_message = Cpt(PytmcSignal, ':ERR:MSG', io='i', kind='omitted')
 
 
+class XPIMLED(Device):
+    """
+    Controllable illumination with auto-on, auto-off, and shutdown timer.
+    """
```

Comment:
```suggestion
    """Controllable illumination with auto-on, auto-off, and shutdown timer."""
```

Reply:
I dislike how one-line docstrings look so much that I'm going to make the docstring longer instead

Reply:
Lmao ok